### PR TITLE
Chore: fix vulnerabilities

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
Upgrade storybook to 6.5.21 and fix the following vulnerabilities: 

- json5 via `npm audit fix`
- loader-utils via `npm audit fix`
- glob parent via `overrides`
- trim-newlines via `overrides`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -98,6 +98,7 @@
   "dependencies": {
     "@inrupt/solid-client": "^1.25.0",
     "@inrupt/solid-client-authn-browser": "^1.13.0",
+    "@storybook/mdx2-csf": "^0.0.3",
```
Comment (NSeydoux) [inline package.json:101]: The commit message introducing this mentions it as a dev dependency, but it appears here as a prod dependency.

File: .storybook/main.js [hunk 2]
```diff
@@ -15,6 +22,9 @@ module.exports = {
       },
     },
   ],
+    features: {
+    previewMdx2: true,
+  },
```
Comment (NSeydoux) [inline .storybook/main.js:27]: Out of curiosity, why was this required?

File: package.json (hunk 1 above)
Comment (VirginiaBalseiro) [reply package.json]: That was a mistake, amended commit.

File: .storybook/main.js (hunk 2 above)
Comment (VirginiaBalseiro) [reply .storybook/main.js]: It is not! It was leftover from experimenting while trying to get storybook to work and forgot, removed now. Thanks!!

File: package.json [hunk 3]
```diff
@@ -106,7 +107,11 @@
...
   },
+  "overrides": {
+    "glob-parent": "5.1.2",
+    "trim-newlines": "3.0.1"
+  },
   "engines": {
     "node": "^14.17.0 || >=16.0.0"
   }
-}
+}
```
Comment (ThisIsMissEm) [inline package.json:117]: do make sure the trailing new line is preserved

**Review by chelseapinka — APPROVED** [review]
